GitUI/CommandsDialogs/FormDiff.Designer.cs

```diff
@@ -418,11 +418,9 @@ private void InitializeComponent()
             // 
             // findInDiffToolStripMenuItem
             // 
-            findInDiffToolStripMenuItem.Image = Properties.Images.Preview;
             findInDiffToolStripMenuItem.Name = "findInDiffToolStripMenuItem";
-            findInDiffToolStripMenuItem.ShortcutKeys = ((Keys)((Keys.Control | Keys.F)));
             findInDiffToolStripMenuItem.Size = new Size(210, 22);
-            findInDiffToolStripMenuItem.Text = "Find";
+            findInDiffToolStripMenuItem.Text = "Find file...";
```

Comment:
```suggestion
            findInDiffToolStripMenuItem.Text = "&Find file...";
```

Reply:
Added mnemonics to the menu. Probably misses, but I wasted some 30 minutes or so on this useless form...